modules/terminate-agent-hook/main.tf

```diff
@@ -18,17 +18,17 @@ data "archive_file" "terminate_runner_instances_lambda" {
 # tfsec:ignore:aws-lambda-enable-tracing
 resource "aws_lambda_function" "terminate_runner_instances" {
   architectures    = ["x86_64"]
-  description      = "Lifecycle hook for terminating GitLab runner instances"
+  description      = "Lifecycle hook for terminating GitLab runner agent instances"
   filename         = data.archive_file.terminate_runner_instances_lambda.output_path
   source_code_hash = data.archive_file.terminate_runner_instances_lambda.output_base64sha256
   function_name    = "${var.environment}-${var.name}"
   handler          = "lambda_function.handler"
-  memory_size      = var.lambda_memory_size
+  memory_size      = 128
   package_type     = "Zip"
   publish          = true
   role             = aws_iam_role.lambda.arn
-  runtime          = var.lambda_runtime
-  timeout          = var.lambda_timeout
+  runtime          = "python3.8"
```

Comment:
can we update this without problems to 3.9?.

Reply:
We should do it, yes. Not sure if everything will run smooth. Created #631 for this point.